### PR TITLE
Des 398 fe create edit recursos not the table

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

**Review by abdulflorez — APPROVED** [review]

File: components/organisms/AddResource/AddResource.module.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+.inputOptions {
+  width: 28rem; 
+  padding: 0.6rem;
+  border: none;
+  border-radius: var(--borderRadius);
+  outline: none;
+  background-color: transparent;
+  font-family: var(--primaryFont);
+  font-size: var(--fontSize-paragraph_mobile);
+  border: 1px solid var(--turquoise);
```
Comment (DanielCalleCO) [inline components/organisms/AddResource/AddResource.module.css:84]: px

File: components/organisms/AddResource/AddResource.module.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+.name__file--select {
+  color: var(--white);
+}
+
+.thumbnail-preview {
+  max-width: 7.5rem; 
+  max-height: 9.375rem; 
+  overflow: hidden;
+  border: 1px solid var(--turquoise);
```
Comment (DanielCalleCO) [inline components/organisms/AddResource/AddResource.module.css:41]: PX

**Review by DanielCalleCO — DISMISSED** [review]

**Review by cristianarce89 — APPROVED** [review]

File: components/organisms/AddResource/AddResource.module.css (hunk 1 above)
Comment (Daniielmayo) [reply components/organisms/AddResource/AddResource.module.css:84]: fixed to rem

File: components/organisms/AddResource/AddResource.module.css (hunk 2 above)
Comment (Daniielmayo) [reply components/organisms/AddResource/AddResource.module.css:41]: fixed to rem

**Review by DanielCalleCO — APPROVED** [review]

**Review by abdulflorez — APPROVED** [review]

**Pull request merged**